tensorflow_datasets/translate/en_tam_parallel_text.py

```diff
@@ -0,0 +1,96 @@
+# # coding=utf-8
+# Copyright 2019 The TensorFlow Datasets Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""English-Tamil parallel text corpus from Morphological Processing 2012"""
+
+from __future__ import absolute_import
+from __future__ import division
+from __future__ import print_function
+
+import os
+import tensorflow as tf
+import tensorflow_datasets.public_api as tfds
+
+
+DESCRIPTION = """\
+	The parallel corpora cover texts from bible, cinema and news domains.
+"""
+CITATION = """\
+    @InProceedings {biblio:RaBoMorphologicalProcessing2012,
+	title     = {Morphological Processing for English-Tamil Statistical Machine Translation},
+	author    = {Loganathan Ramasamy and Ond{\v{r}}ej Bojar and Zden{\v{e}}k {\v{Z}}abokrtsk{\'{y}}},
+	year      = {2012},
+	pages     = {113--122},
+	Booktitle = {Proceedings of the Workshop on Machine Translation and Parsing in Indian Languages ({MTPIL}-2012)},
+}
+"""
+
+
+
+class EnTamParallelText(tfds.core.GeneratorBasedBuilder):
+  """(en_tam_parallel_text): English_Tamil parallel text corpus"""
+  VERSION = tfds.core.Version(
+            "0.0.3", experiments={tfds.core.Experiment.S3: False})

```

Comment:
`{tfds.core.Experiment.S3: False}` is the deprecated legacy mode. Please remove.
You'll have to update the `_generate_examples` method to `yield key, example` instead.
See the doc for more info: https://www.tensorflow.org/datasets/add_dataset#writing_an_example_generator